src/Particular.CodeRules/Particular.CodeRules.csproj

```diff
@@ -1,8 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard1.1</TargetFramework>
-    <PackageTargetFallback>portable-net45+win8</PackageTargetFallback>
```

Comment:
To address this new violation:

> error NU1003: PackageTargetFallback and AssetTargetFallback cannot be used together. Remove PackageTargetFallback(deprecated) references from the project environment.

I don't fully understand it, since we don't explicitly use `AssetTargetFallback` anywhere.

Reply:
I believe `AssetTargetFallback` is used in SDK targets files. Regardless, when moving to `netstandard2.0`, it's correct to remove the `PackageTargetFallback` entry.